dashboard-app/src/components/assets/logo_image.svg

```diff
@@ -0,0 +1,21 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="84.956" height="84.868" viewBox="0 0 84.956 84.868">
```

Comment:
Also you should probs add "assets" to the code climate exclude list (in repo settings) to avoid it saying the coverage has gone down.